examples/colab/image_enhancing.ipynb

```diff
@@ -0,0 +1,444 @@
+{
```

Comment:
<p>Again, please don't do saving inside plotting. It can be done outside and have each function do its own specialty.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/hub/pull/352/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>okay</p>